openrouteservice/src/main/java/org/heigit/ors/api/requests/routing/RouteRequestHandler.java

```
@@ -504,4 +512,13 @@ private  boolean convertSetFlexibleMode(boolean useContractionHierarchies) throw
 
         return avoidCountryIds;
     }
+
+    private double convertUserSpeed(Double userSpeed) throws ParameterValueException {
+        double maxSpeed = ((AppConfig.getGlobal().getServiceParameter("routing.profiles.default_params", "maximum_speed")) != null) ? Double.parseDouble(AppConfig.getGlobal().getServiceParameter("routing.profiles.default_params", "maximum_speed")) : 80; //Minimum speed of the core.
+        if (userSpeed < maxSpeed) {
+            throw new ParameterValueException(RoutingErrorCodes.INVALID_PARAMETER_VALUE, RouteRequest.PARAM_USER_SPEED);
```

Could we return an error message containing the value set by the user together with an information such as "The maximum speed must not be lower than 80km/h" where the value should reflect the actually configured limit?